R/fmelt.R

```diff
@@ -8,14 +8,15 @@ melt <- function(data, ..., na.rm = FALSE, value.name = "value") {
     UseMethod("melt", data)
     # if data is not data.table and reshape2 is installed, this won't dispatch to reshape2's method;
     # CRAN package edarf and others fail without the else branch
+  # nocov start
   } else {
-    # nocov start
+    data_name = deparse(substitute(data))
```

Comment:
I would avoid deparse if not really needed, as here. For a very long symbol names it can cause problems. The gain from using that here is really minor.